test/fixtures/user-integration-app/common/models/post.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "post",
+  "forceId": false,
```

Comment:
AFAICT, we are not setting `Post.id` explicitly in any of the existing tests. Why is it necessary to set `forceId` to `false` in that case?